entries/p/ptv.vic.gov.au.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Public Transport Victoria": {
+    "domain": "ptv.vic.gov.au",
+    "contact": {
+      "facebook": "publictransportvictoria",
```

Comment:
The facebook page link needs to go as the page is restricted.
```suggestion
```

Reply:
This is strange because I can access the Facebook page and I don't have a Facebook account. Maybe it's location based?

Reply:
Location could be one of the restrictions. All I know is neither I nor CircleCI can access the Facebook page so it has to be removed for the build to succeed.